changedetectionio/babel.cfg

```diff
@@ -0,0 +1 @@
+[jinja2: **.html]
```

Comment:
To support i8n for py file, `[python: **.py]`. But there is some settings I didn't figure out, I do it later.